yt/frontends/ramses/tests/test_outputs.py

```diff
@@ -683,3 +684,104 @@ def _dummy_field(field, data):
     v1 = ad["gas", "test"]
 
     np.testing.assert_allclose(v0, v1)
+
+
+# Simple context manager that overrides the value of
+# the self_shielding flag in the namelist.txt file
+@contextmanager
+def override_self_shielding(fname: str, section: str, val: bool):
```

Comment:
seems pretty unlikely it'd happen and cause issues, but this does introduce a race condition -- since this manager actually overwrites the existing namelist file, if tests were run in parallel then other tests using `ramses_new_fromat` could read in the overwritten namelist unintentionally (or catch the file during the active write?). It is a small enough dataset (4.4 MB) that you could instead copy the whole dataset into a temporary directory and modify the namelist files appropriately (which would be cleaner to do with pytest over in `test_outputs_pytest.py`). Would take a bit of extra IO but IMO worth it for the extra future-proofing, but also a small enough thing that I could be convinced that I'm overthinking this :)

Reply:
This should be good now :)